### DISCUSSION TITLE
Raycast as a go to replacement for Spotlight

### DISCUSSION THREAD

**Discussion by mikoscz** [Selleo/commons#9] [Ideas]
https://www.raycast.com/

I use it for most of the system navigation, shortcuts, quick docs lookup, StackOverflow, meetings, schedules and many more.

**Comment by qbart** [comment]
is it possible to script it? and share extensions?

**Reply by mikoscz** [reply]
You can script it, create extensions, add shortcuts for bash scripts etc. For public extensions - yes, for private it requires paid plan, but if you skip the store everyone can install it manually through GH.

**Reply by qbart** [reply]
i was imprecise, can you script the installation? so anyone who runs `setup.sh` can install raycast?
maybe we should also cretae a list of Apps that are recommended

**Reply by mikoscz** [reply]
`brew --cask` would do the job

**Comment by k1eu** [comment]
+1 on this one, except of installation later on we can add company wide scripts like workloggin or something else :)

**Reply by k1eu** [reply]
I had kudo distribution script for the Raycast somewhere
